applications/incompressible_flow_with_transport/rising_bubble/application.h

```diff
@@ -226,7 +226,7 @@ class Application : public FTI::ApplicationBase<dim, Number>
 
     // MATHEMATICAL MODEL
     param.problem_type                = ProblemType::Unsteady;
-    param.equation_type               = EquationType::Convection;
+    param.equation_type               = EquationType::ConvectionDiffusion;
```

Comment:
due to PR #496, one should be able to use `Convection` only as in the previous code version. I will change this back to the original version.